setup.py

```diff
@@ -1,18 +1,16 @@
 #!/usr/bin/env python
 
-from distutils.core import setup
-setup(name='defectMOF',
+from setuptools import setup, find_packages
+
+setup(name='defects',
       version='0.1.0',
       description='Introducing Missing-linker Defects in Metal-Organic Frameworks',
       author='Meiirbek Islamov',
       author_email='mei12@pitt.edu',
       url='https://github.com/meiirbek-islamov/defects',
-      packages=['defectMOF'],
+      packages=find_packages(include=['defects']),
```

Comment:
this is correct, but should be squashed with the commit where you added the setup.py, since that was clearly incorrect.

Also, this commit "remove random in requirments.txt" contains a bunch of things that should be separated out that don't have to do with removing random.

You delete the test/.travis.yml file, so that should be squashed with where you added it, since it shouldn't have been added in the first place.

__pycache__ files also need to be added to your .gitignore and shouldn't be committed to the repo. 

In general, don't commit things to the repo unless you wrote them, or are SURE they should be in the repo!